### PR TITLE
Patch dropdown trigger leading

### PR DESCRIPTION
**Pull request by philschanely**
## Description

This PR makes a quick patch to dropdown trigger styles that ensure consistent size and alignment for the button trigger.

## Testing in `sage-lib`

See no changes in Components > Dropdown

## Testing in `kajabi-products`

1. (LOW) Fix to line height in dropdown button trigger fixes tiny alignment issue.

## Related

#489

### CODE REVIEW

File: packages/sage-assets/lib/stylesheets/components/_dropdown.scss [hunk 1]
```diff
@@ -336,16 +336,6 @@ $-dropdown-trigger-dropdown-offset: rem(16px) + sage-spacing(xs);
   &.sage-dropdown__trigger--options {
     min-width: auto;
   }
-
-  > .sage-btn {
-    line-height: sage-line-height(md);
```
Comment (philschanely) [inline packages/sage-assets/lib/stylesheets/components/_dropdown.scss:341]: @monicawheeler just double-checking here. This moved to `_button.scss` in this PR but also changed to `body` for the parameter. If `md` is desired then we need to also set the second paramenter to `false`. I've clarified the documentation of this function as well.

Comment (philschanely) [reply packages/sage-assets/lib/stylesheets/components/_dropdown.scss:341]: BTW this was traced to #489

File: packages/sage-assets/lib/stylesheets/tokens/_line_height.scss [hunk 2]
```diff
@@ -48,8 +48,30 @@ $sage-line-heights: (
...
 /// Sage line height token utility
 ///
-/// @param {string} $key [body] The token to retrieve
-/// @param {bool} $responsive [true] Whether or not retrieve a responsive css custom property or a flat value
+/// @param {string} $key [body] The token to retrieve.
+///    By default, this should be a type spec token.
+///    If the next parameter is `false` then this should be a line height token.
+/// @param {bool} $responsive [true] Whether or not retrieve a responsive css custom property (based on type specs) or a flat value (based on line-height tokens).
+///
+/// @example scss - Responsive setting
```
Comment (monkeypox8) [inline packages/sage-assets/lib/stylesheets/tokens/_line_height.scss:56]: 😍

**Review by monkeypox8 — APPROVED** [review]
👍🏼 LGTM

**Pull request merged**